### PR TITLE
Add documentation for the Enhanced Consent JS tracker plugin

### PR DESCRIPTION
**Pull request by igneel64**
Adding the Enhanced Consent plugin documentation after its release on the Marketing Consent accelerator.

### CODE REVIEW

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/enhanced-consent/index.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+### Consent Entity
+
+An consent entity can have the following attributes:
+
+| attribute | type | description | required |
+| :--------------: | :------: | :----------------------------------------------------------------------------------------------------------------: | :------: |
+| basisForProcessing | `string` | The type of the page that was visited E.g. homepage, product page, checkout page. | ✅ |
+| consentUrl | `string` | The language that the web page is based in. | ✅ |
+| consentVersion | `string` | The locale version of the site that is running. | ✅ |
+| consentScopes | `string[]` | The locale version of the site that is running. | ✅ |
```
Comment (matus-tomlein) [inline docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/enhanced-consent/index.md:195]: The descriptions for the following three fields do not match them.

File: docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/enhanced-consent/index.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+### Consent Entity
+
+An consent entity can have the following attributes:
+
+| attribute | type | description | required |
+| :--------------: | :------: | :----------------------------------------------------------------------------------------------------------------: | :------: |
+| basisForProcessing | `string` | The type of the page that was visited E.g. homepage, product page, checkout page. | ✅ |
+| consentUrl | `string` | The language that the web page is based in. | ✅ |
+| consentVersion | `string` | The locale version of the site that is running. | ✅ |
+| consentScopes | `string[]` | The locale version of the site that is running. | ✅ |
```
Comment (matus-tomlein) [inline docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/enhanced-consent/index.md:180]: Same as before – need to update the descriptions here.

**Review by matus-tomlein — APPROVED** [review]
Other than the missing descriptions in the table, it LGTM!

**Comment by igneel64** [comment]
@matus-tomlein Yeap didn't save the files after the edit 🤦  Updating

**Pull request merged**